README.md

```diff
@@ -1,24 +1,28 @@
-# Aurora-BP Study Data
+# The Aurora-BP Dataset and Study

```

Comment:
An ordering of "The Aurora-BP Study and Dataset" seems to make more chronological sense. Did you have a preference for this order ("... Dataset and Study") instead?

Reply:
sure, sounds good to me! :) I'll make the change and push